File: app/Model/Vote.ts
//     Copyright (c) Microsoft Corporation.  All rights reserved.

/// <reference path="../references/index.d.ts" />
/// <reference path="BeerInfo.ts" />

module DXLiquidIntel.App.Model {

    export class Vote {
        public VoteId: number
        public PersonnelNumber: number
        public VoteDate: Date
        public UntappdId: number
        public BeerName?: string
        public Brewery?: string
        public BeerInfo?: BeerInfo
    }

    export class VoteTally {
        public UntappdId: number
        public BeerName?: string
        public Brewery?: string
        public VoteCount: number
    }
}
         

File: mobile/xamarin/SmartKegerator/SmartKegerator/Models/Vote.cs
using System;
namespace SmartKegerator.Models
{
    public class Vote
    {
        public int VoteId { get; set; }
		public int PersonnelNumber { get; set; }
		public DateTime VoteDate { get; set; }
		public int UntappdId { get; set; }
		public string BeerName { get; set; }
		public string Brewery { get; set; }
		public int untappdId { get; set; }
		public string name { get; set; }
		public string beer_type { get; set; }
		public double ibu { get; set; }
		public double abv { get; set; }
		public string description { get; set; }
		public string brewery { get; set; }
		public string image { get; set; }
    }
}
